docs/advanced.rst

```diff
@@ -75,53 +75,6 @@ to explicitly specify property :php:attr:`Connection::query_class`::
 This is also useful, if you have created your own Query class in a different
 namespace and wish to use it.
 
-Using Debug Proxy Connections (Stopwatch and Profiler)
```

Comment:
Why are we dropping Stopwatch and Profiler proxy connections?

Reply:
Because they are wrongly designed:
- if driverType is/was used somewhere, it was meant to be used to check the underlaying DB type, ie. with debug connections it will not work
- in the future, we should replace connections classes with DBAL connection classes which support debug hooks which is the correct way to achive that debug support (now, you can implement it in a few lines on your own)

we should remove Proxy connection as well and simplify dsql even more (likie nested execute - we should not really care about it)

Reply:
I am fine with this as long as there is way to debug

Reply:
please approve if ok, RTM, I will create now a PR for data to adjust the code

keeping ProxyConnection for now